mmaction/integration/nncf/compression.py

```diff
@@ -0,0 +1,288 @@
+import os
+import pathlib
+import tempfile
+from functools import partial
+
+import mmcv
+import torch
+
+from mmaction.utils import get_root_logger
+from .utils import (check_nncf_is_enabled, get_nncf_version, is_nncf_enabled,
+                    load_checkpoint, no_nncf_trace)
+
+SHOULD_DBG = False
+
+def print_dbg(*args, **kwargs):
+    if SHOULD_DBG:
+        print(*args, **kwargs)
+
+if is_nncf_enabled():
+    try:
+        from nncf.torch.initialization import DefaultInitializingDataLoader
+        from nncf.torch.nncf_network import NNCFNetwork
+        
+        class_InitializingDataLoader = DefaultInitializingDataLoader
+    except ImportError:
+        raise RuntimeError(
+            'Cannot import the standard functions of NNCF library '
```

Comment:
Why do you check it during the module loading instead of directly checking when it's necessary?